### PR TITLE
Update SkiaSharp to the latest stable version (2.88.6)

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change

Time to get new things!

### CODE REVIEW

**Comment by brminnick** [comment]
Thanks @mattleibow! Any estimation for when this PR will be released?

As I'm sure you're aware, a [security vulnerability was reported on `SkiaSharp` this weekend](https://github.com/advisories/GHSA-j7hp-h8jx-5ppr). The good news is that the vulnerability  is patched in `SkiaSharp v2.88.6`, the version this PR updates to for .NET MAUI, so the next MAUI release that includes this PR resolves this security vulnerability for .NET MAUI devs! 

There's many libraries that use `Microsoft.Maui.Graphics.Skia`, and the latest stable version of `Microsoft.Maui.Graphics.Skia` currently uses `SkiaSharp v2.88.2` (includes security vulnerability), and the latest preview version of `Microsoft.Maui.Graphics.Skia` currently uses `SkiaSharp v2.88.4-preview.82` (also includes security vulnerability).

The biggest headache I've discovered so far is that `tizen-net7.0` library has a transitive dependency to SkiaSharp via `net7.0-tizen` -> `Tizen.UIExtensions.NUI` -> `Microsoft.Maui.Graphics.Skia` -> `SkiaSharp`. @JoonghyunCho on the Tizen team is aware of it and will be working with his team to bump `Tizen.UIExtensions.NUI`'s dependency to `Microsoft.Maui.Graphics.Skia` once the new version of `Microsoft.Maui.Graphics.Skia` containing this PR has been released.

The good news is that we've patched it in `CommunityToolkit.Maui` by adding a direct dependency to `SkiaSharp v2.88.6` for our Tizen builds, and we will be releasing `CommunityToolkit.Maui v6.0.0` this week. So any dev who updates to `CommunityToolkit.Maui v6.0.0` will by default be using the latest version of `SkiaSharp` on Tizen, alleviating any potential security vulnerabilities for them. I've also [created an Issue to remind us to remove this direct dependency to `SkiaSharp` once the new version of `Microsoft.Maui.Graphics.Skia` is released](https://github.com/CommunityToolkit/Maui/issues/1424).

tl;dr There's security vulnerability in SkiaSharp, patched in `SkiaSharp v2.88.6`, and `Microsoft.Maui.Graphics.Skia` is the long-tail that needs to be patched (done) + released (pending) first before other libraries can fix their transitive vulnerabilities.

**Review by rmarinho — APPROVED** [review]

**Comment by mattleibow** [comment]
Backport in https://github.com/dotnet/maui/pull/17663
